### PR TITLE
[16.0][MIG] impersonate_login: backport from V17 to V16

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Review by CasVissers-360ERP — APPROVED** [review]

**Comment by CasVissers-360ERP** [comment]
Tested / functional review.

File: impersonate_login/models/impersonate_log.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+from odoo import fields, models
+
+
+class ImpersonateLog(models.Model):
+    _name = "impersonate.log"
+    _description = "Impersonate Logs"
+
+    user_id = fields.Many2one(
+        comodel_name="res.partner",
```
Comment (Kev-Roche) [inline impersonate_login/models/impersonate_log.py]:
```suggestion
        comodel_name="res.users",
```

**Review by Kev-Roche — CHANGES_REQUESTED** [review]
@astirpe, sincerely thank you to bring this module alive and fix it from the PR in V14 !
I have a question, in my original PR, I had css to visually show when we are logged with an impersonate user. Why not keep it ?

File: impersonate_login/models/impersonate_log.py (hunk 1 above)
Comment (astirpe) [reply impersonate_login/models/impersonate_log.py]: Thank you!

**Comment by astirpe** [comment]
@Kev-Roche Thank you!
Porting the css needed also the porting of the rest of the js logic, it was taking too much time to me. I was more focused to make this module just working in V17. But I think the css part is an important usability feature. Is it maybe something you can implement?

**Review by Kev-Roche — APPROVED** [review]

**Comment by Kev-Roche** [comment]
> @Kev-Roche Thank you! Porting the css needed also the porting of the rest of the js logic, it was taking too much time to me. I was more focused to make this module just working in V17. But I think the css part is an important usability feature. Is it maybe something you can implement?

Ok, I'll work on this part.

**Review by rvalyi — APPROVED** [review]

**Pull request merged**